Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM alpine:3.5
+FROM vertigo/go-builder as builder
```

Comment:
We won't accept a non official build base image btw. Our solution was to use the official go builder image + install anything missing on top of it.